composer.lock

```diff
@@ -816,16 +816,16 @@
         },
         {
             "name": "nikic/php-parser",
-            "version": "v4.3.0",
+            "version": "v4.4.0",
```

Comment:
We do test with the lowest dependencies on Travis, so I'm surprised why just fixing the lock file helps. The build would fail when I tried to update PHPUnit to `^9.1` because of a bug in PHPUnit 9.1.0. Requiring `^9.1.1` solved the problem.

Depending on the exact nature of the issue, should we introduce a constraint or we bump/adjust some other constraint that would eliminate the conflict?

Reply:
> We do test with the lowest dependencies on Travis, so I'm surprised why just fixing the lock file helps.

This helps fix a crash I had when running Psalm, and we don't run Psalm after running `composer --prefer-lowest`

I think we should not use a conflict when it's something that fails in dev/CI, I can add a `require-dev` constraint if you want me to

Reply:
Agree. Let's do a temporary `require-dev`. Additionally, is it worth reporting to Psalm so that they could fix it on their end and we could remove this later?

Reply:
Requirement added, bug reported here: https://github.com/vimeo/psalm/issues/3155